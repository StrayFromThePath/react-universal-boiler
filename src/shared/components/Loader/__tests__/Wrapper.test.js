import React from 'react';
import renderer from 'react-test-renderer';

import Wrapper from '../Wrapper';

jest.useFakeTimers();
Date.now = jest.fn(() => 1482363367071);

it('renders and matches the snapshot', () => {
  const tree = renderer.create(<Wrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
