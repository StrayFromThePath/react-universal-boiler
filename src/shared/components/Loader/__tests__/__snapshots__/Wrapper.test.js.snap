// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders and matches the snapshot 1`] = `
<div
  className="sc-bdVaJa kAPAHy"
/>
`;
